package.json

```diff
@@ -7,10 +7,11 @@
   "react-native": "dist-native/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc --emitDeclarationOnly && BROWSER=1 parcel build src/index.ts",
+    "build": "tsc && BROWSER=1 parcel build src/index.ts",
```

Comment:
How does this work? I think we should not put files compiled by tsc in the resulting package, and we had `emitDeclarationOnly` to achieve it.

Reply:
Got it, restored how it was 👍